src/inc/corprof.idl

```diff
@@ -2408,6 +2412,31 @@ interface ICorProfilerCallback8 : ICorProfilerCallback7
         [in] BOOL       fIsSafeToBlock);
 }
 
+[
+    object,
+    uuid(27583EC3-C8F5-482F-8052-194B8CE4705A),
+    pointer_default(unique),
+    local
+]
+interface ICorProfilerCallback9 : ICorProfilerCallback8
+{
+    // This event is triggered whenever a dynamic method is garbage collected
+    // and subsequently unloaded.
+
+    HRESULT DynamicMethodUnloaded([in] FunctionID functionId);
+
+    HRESULT EventPipeEventDelivered(
+        [in] REFGUID eventGuid,
```

Comment:
@vancem if Brian will make this a string, can it be his change that does it all? Or are you suggesting I just make the GUID a string regardless of his change?